### PR TITLE
Help text for Form components always shows (like Parcel Ids on submission intake)

### PR DESCRIPTION
**Pull request by qhanson55**
# Description

Help text on form components like InputText now always displays
so that the user can see the help text without needing to clicking into field, saving confusion
[PADS-245](https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-245)

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/components/form/InputText.vue [hunk 1]
```diff
@@ -51,7 +53,7 @@ const fieldActive: Ref<boolean> = ref(false);
     />
 
     <small
-      v-if="fieldActive"
+      v-if="fieldActive || true"
```
Comment (wilwong89) [inline frontend/src/components/form/InputText.vue:56]: I think you wanted `true` to actually be `helpVisible`.

File: frontend/src/components/form/InputText.vue [hunk 2]
```diff
@@ -51,7 +53,7 @@ const fieldActive: Ref<boolean> = ref(false);
     />
 
     <small
-      v-if="fieldActive"
+      v-if="fieldActive || helpVisible"
       :id="`${name}-help`"
     >
```
Comment (kyle1morel) [inline frontend/src/components/form/InputText.vue:58]: We should probably figure out a common functionality across all our form components. Currently all of them except `InputText` show the help by default. This is the only one with multiple diverging methods.

**Review by kyle1morel — APPROVED** [review]
Approved. Please push an update to remove the unused imports if you can. Always a good idea to run `npm run lint` before commits.

**Comment by qhanson55** [comment]
> Approved. Please push an update to remove the unused imports if you can. Always a good idea to run `npm run lint` before commits.

done

**Pull request merged**